webclient/app/src/app/commons/CockpitAppBar.jsx

```diff
@@ -21,10 +31,17 @@ function CockpitAppBar() {
                             <HomeIcon />
                         </IconButton>
                         <Typography variant="h1" component="div" sx={{flexGrow: 1}}>KI Cockpit</Typography>
```

Comment:
Please add translations for KI Cockpit (AI Cockpit) and "Auf Karte anzeigen" (Show on Map)